oidc.go

```diff
@@ -192,6 +193,18 @@ type UserInfo struct {
 	claims []byte
 }
 
+// userInfoRaw allows us to decode both compliant and non-compliant Cognito userInfo
+// Cognito returns email_verified as a string instead of as a bool.
+// This is not documented by AWS as far as I can tell, but see for example
```

Comment:
nit: Comments generally aren't in the first person.

Maybe just a comment above EmailVerified

```go
type userInfoRaw struct {
	Subject       string       `json:"sub"`
	Profile       string       `json:"profile"`
	Email         string       `json:"email"`
	// Handle providers that return email_verified as a string.
	// https://forums.aws.amazon.com/thread.jspa?messageID=949441&#949441
	// https://discuss.elastic.co/t/openid-error-after-authenticating-against-aws-cognito/206018/11
	EmailVerified stringAsBool `json:"email_verified"`
}
```

Reply:
I deleted the comments above and added the comments above EmailVerified.